### PR TITLE
v0.1.7: Fix several bugs and make several improvements

### PR DESCRIPTION
**Pull request by HollowMan6**
We shall get the following PRs merged before we make v0.1.7 release:
- #14 
- #15 

This PR resolves:
- #16 
- #17 
- #18 

It also allows users to auto-focus the workspace so we don't need to click on the workspace first to make the selection when we have multiple workspaces at the same time.

### CODE REVIEW

**Comment by HollowMan6** [comment]
cc: @ewpatton

File: src/multiselect.js [hunk 1]
```diff
@@ -42,12 +42,16 @@ export class Multiselect {
...
+        injectionDiv, 'keydown', this, this.onKeyDown_);
     this.onKeyUpWrapper_ = Blockly.browserEvents.conditionalBind(
-        globalThis['window'], 'blur', this, this.onBlur_);
-    this.onKeyUpWrapper_ = Blockly.browserEvents.conditionalBind(
-        this.workspace_.getInjectionDiv(), 'keyup', this, this.onKeyUp_);
+        injectionDiv, 'keyup', this, this.onKeyUp_);
+    this.onFocusOutWrapper_ = Blockly.browserEvents.conditionalBind(
+        injectionDiv, 'focusout', this, this.onBlur_);
+    injectionDiv.addEventListener('mouseenter', () => {
+      this.workspace_.svgGroup_.parentElement.focus();
```
Comment (ewpatton) [inline src/multiselect.js:53]: Is there a public accessor for the svgGroup_ field? The field may be obfuscated in a production build.

File: src/multiselect_controls.js [hunk 2]
```diff
@@ -348,10 +348,11 @@ export class MultiselectControls {
...
     }
     this.dragSelect_ = new DragSelect({
-      selectables: document.querySelectorAll(
-          'g.blocklyDraggable:not(.blocklyInsertionMarker)' +
+      selectables: this.workspace_.getInjectionDiv()
+          .querySelectorAll(
+              'g.blocklyDraggable:not(.blocklyInsertionMarker)' +
         '> path.blocklyPath'),
-      area: document.querySelector('.blocklyWorkspace'),
+      area: this.workspace_.svgGroup_,
```
Comment (ewpatton) [inline src/multiselect_controls.js:355]: Is there a public accessor for the svgGroup_ field? The field may be obfuscated in a production build.

**Review by ewpatton — CHANGES_REQUESTED** [review]

File: src/multiselect.js (hunk 1 above)
Comment (HollowMan6) [reply src/multiselect.js:53]: Unfortunately, it seems like they don't have a public accessor for the `svgGroup_` field, and I can see there are also some use in blockly's official plugins:

https://github.com/google/blockly-samples/blob/46dce3f6031c7bb906e992767c17951a91dd1f6d/plugins/scroll-options/src/index.js#L82

Comment (HollowMan6) [reply src/multiselect.js:53]: Issue for this: https://github.com/google/blockly/issues/7157

**Comment by laurensvalk** [comment]
Thanks both for such a quick review and fixes. I'll give this a go next week and keep you posted.

**Review by ewpatton — APPROVED** [review]

**Pull request merged**